IHP/ErrorController.hs

```diff
@@ -48,14 +47,14 @@ handleNoResponseReturned controller = do
             
         |]
     let title = [hsx|No response returned in {tshow controller}|]
-    let (RequestContext _ respond _ _ _) = ?requestContext
+    let (RequestContext _ respond _ _ _ _) = ?requestContext
```

Comment:
Maybe when we're on this already we could replace this with a named pattern match:
```suggestion
    let RequestContext { respond } = ?requestContext
```

This it's more robust against future changes when we want to add more fields :)